### PR TITLE
Ohio HL7 Change Requests

### PR DESCRIPTION
**Pull request by MauriceReeves-usds**
This PR makes some small modifications to how Ohio's HL7 is generated, and adds a new naming format for them.

## Changes
- Adds a naming format for Ohio
- Adds an enum for naming format types
- Updates the Ohio schema to suppress a bareword they can't process
- Updates the translation to pass in flag for QST suppression
- Sync's the organization files

## Checklist
- [x] Tested locally?
- [x] Ran `quickTest all`?
- [x] Ran `./prime test` against local Docker ReportStream container?
- [x] Downloaded a file from http://localhost:7071/api/download
- [ ] Updated the release notes? 
- [ ] Added tests?
- [ ] Did you check for sensitive data, and remove any? 
- [ ] Does logging contain sensitive data?  
- [ ] Are additional approvals needed for this change?
- [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?

## Security
*List potential security threats and mitigations in the PR*

## Fixes
*List GitHub issues this PR fixes*
- #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

### CODE REVIEW

File: prime-router/settings/organizations-prod.yml [hunk 1]
```diff
@@ -156,7 +156,9 @@
     - name: elr
       organizationName: nd-doh
       topic: covid-19
-      jurisdictionalFilter: [ "matches(ordering_facility_state, ND)" ]
+      jurisdictionalFilter:
+        - matches(ordering_facility_state, ND)
+      deidentify: false
```
Comment (jimduff-usds) [inline prime-router/settings/organizations-prod.yml:161]: To be safe, I think we should change the default for this to be deidentify: true   if not specificied.

File: prime-router/settings/organizations-prod.yml [hunk 2]
```diff
@@ -240,7 +246,8 @@
     - name: elr
```
Comment (jimduff-usds) [inline prime-router/settings/organizations-prod.yml:246]: NM missing transport and timing, but I assume that's coming soon

File: prime-router/settings/organizations-prod.yml [hunk 3]
```diff
@@ -258,7 +265,8 @@
...
```
Comment (jimduff-usds) [inline prime-router/settings/organizations-prod.yml:265]: MT missing transport and timing, but I assume that's coming

File: prime-router/src/main/kotlin/Report.kt [hunk 4]
```diff
@@ -39,6 +39,12 @@ const val REPORT_MAX_ERRORS = 100
  * unique id and name as well as list of sources for the creation of the report.
  */
 class Report {
+    enum class NameFormat {
+        STANDARD,
+        APHL,
+        OHIO,
+    }
```
Comment (jimduff-usds) [inline prime-router/src/main/kotlin/Report.kt:46]: If this continues to grow, might want to refactor into its own full-blown class, and pattern its style similar to 'Mappings' and 'JurisdictionFilters'.   While we're at it, we should rename Mapping to something less confusingly generic, like ValueMapping, FieldMapping, or ElementValueMapping or somesuch.

File: prime-router/src/main/kotlin/Report.kt [hunk 5]
```diff
@@ -684,39 +690,46 @@ class Report {
...
+                        RO - receiving organization
+                        SE - sending environment (test/prod)
+                        RE - receiving environment (test/prod)
+                        OF - original file name (optional)
+                        Timestamp - creation ts of the file
+                        Extension - HL7 for hl7, csv for csv, etc
+
+                        Examples:
+                        OchsnerHealth_OchsnerHealth_LAOPH_Prod_Test_ORURO112345_20200415082416800.HL7
+                        ChristusHealth_CCS_LAOPH_Prod_Test_20200415082416800.HL7
```
Comment (jimduff-usds) [inline prime-router/src/main/kotlin/Report.kt:716]: I definitely appreciate the extended explanation and especially the examples.

File: prime-router/settings/organizations-local.yml [hunk 6]
```diff
@@ -365,6 +368,62 @@
           initialTime: 00:00
           timeZone: EASTERN
 
+  - name: gu-doh
+    description: Guam Department of Health
```
Comment (jimduff-usds) [inline prime-router/settings/organizations-local.yml:372]: GU and VT are in organizations-local but not in organizations-prod.  By design, I presume.

**Review by jimduff-usds — APPROVED** [review]
The NameFormat is a good and necessary improvement.

As usual I noted things in organizations-*.yml that might or might not be errors; just ignore if things are ok.

File: prime-router/src/main/kotlin/Report.kt (hunk 4 above)
Comment (MauriceReeves-usds) [reply prime-router/src/main/kotlin/Report.kt:46]: I agree. We need to elevate this into something like Mappers, and we probably want to allow a file name to have multiple mapper values in it. Something like:

```
fileName:
    separator: _
    parts:
        - CDCPRIME
        - toTimeStamp('yyyyMMddHHmmss')
```

**Pull request merged**